sky/utils/cli_utils/status_utils.py

```diff
@@ -208,10 +208,17 @@ def _get_zone(cluster_status):
 
 
 def _get_autostop(cluster_status):
-    autostop_str = '-'
+    autostop_str = ''
+    separtion = ''
     if cluster_status['autostop'] >= 0:
         # TODO(zhwu): check the status of the autostop cluster.
         autostop_str = str(cluster_status['autostop']) + ' min'
```

Comment:
Nit: "30 min (down)" can be longer than `AUTOSTOP` column header. Wdyt "30m (down)" / "30m" instead? Not feeling strongly.

Reply:
Good point! Changed. Thanks!

Reply:
Here is the new status table. Wdyt?
```
NAME             LAUNCHED        RESOURCES             STATUS  AUTOSTOP   COMMAND                         
test-autodown    a few secs ago  1x GCP(n1-highmem-8)  UP      1m (down)  sky launch -c test-autodown...  
smoke-test-zhwu  9 hrs ago       1x GCP(n1-highmem-8)  UP      -          sky start smoke-test-zhwu 
```